lib/smooth/core/network/Wifi.cpp

```diff
@@ -119,7 +109,7 @@ namespace smooth::core::network
 
     bool Wifi::is_connected_to_ap() const
     {
-        return connected_to_ap;
+        return connected;
```

Comment:
Now that Wifi inherits from `NetworkInterface` this method is obsolete and replaced by is_connected() so lets remove it.